src/extension/features/accounts/manage-payees/index.js

```diff
@@ -0,0 +1,37 @@
+import { Feature } from 'toolkit/extension/features/feature';
+
+export class ManagePayees extends Feature {
+  observe = changedNodes => {
+    if (
+      changedNodes.has(
+        'ynab-u modal-popup modal-account-edit-transaction-list ember-view modal-overlay active'
+      )
+    ) {
+      this.invoke();
+    }
+  };
+
+  invoke = () => {
```

Comment:
prefer to see `observe()` and `invoke()` as declarations instead of expressions -- if you wouldn't mind updating these :D

Reply:
Sure thing. Directly copied from [clear-selection](https://github.com/toolkit-for-ynab/toolkit-for-ynab/blob/master/src/extension/features/accounts/clear-selection/index.js#L28), will update.